src/utils/featheryClient/integrationClient.ts

```diff
@@ -72,10 +80,12 @@ export default class IntegrationClient {
     };
     return fetch(url, options)
       .then(async (response) => {
-        if (parseResponse) await this._checkResponseSuccess(response);
+        if (parseResponse) await checkResponseSuccess(response);
         return response;
       })
       .catch((e) => {
+        // Throw error for offline handler
+        if (propogateNetworkErrors && e instanceof TypeError) throw e;
```

Comment:
const ignore = this.ignoreNetworkErrors?.current || TYPE_MESSAGES_TO_IGNORE.includes(e.message);
if (ignore && !propagateNetworkErrors && e instanceof TypeError) return;
throw e;